tests/Test.MSAL.NET.Unit/RequestsTests/InteractiveRequestTests.cs

```diff
@@ -135,6 +137,7 @@ public void NoCacheLookup()
             Assert.AreEqual(result.AccessToken, "some-access-token");
 
             Assert.IsTrue(HttpMessageHandlerFactory.IsMocksQueueEmpty, "All mocks should have been consumed");
+            Assert.IsTrue(_myReceiver.EventsReceived.Count > 0);
```

Comment:
Definitely agree. I mean to have more thorough check here, but just do not have a clear idea on how to do that. Meanwhile, I've manually run this test case(s), and with the current test helper `MyReceiver()`, I can actually see all those events on my console. So that means at least the major code structure and logic are there. Just don't pause your review only because that test case thing.